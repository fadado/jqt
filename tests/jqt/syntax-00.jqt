=======================
Syntax tests
=======================

_______________________
Comments
_______________________

1 prefix{# comment #}suffix
2 prefix {# comment #} suffix
3 prefix {# comment #} suffix{# comment #}
4 {# comment #}prefix {# comment #} suffix{# comment #}
5 {# MULTI
	LINE #}
6 prefix{# MULTI
	LINE #}suffix
7 prefix{# MULTI
	LINE #}suffix{# comment #}
8 {# comment #}prefix{# MULTI
	LINE #}suffix{# comment #}
9 {#
	{{ x }}
	{% b %}
	{% b %}xxxxxxxxxxxx
	{% end %}
	{{
	x
	x
	}}
#}

_______________________
Expressions
_______________________

2 + 2 = {{ 2+2 }}
2 + 2 = {{ 2
	   +
	   2 }}
{{ 2
}} + {{ 2
}} = 4

_______________________
One line blocks
_______________________

{% 33 %}1 Diga {{.}}
{% 33 %}2 Diga {{
	.
   }}
{% 33 %}3 Diga {{
	.+1-1
}}
{% 33 %}4 Di{{"ga"}} {{
	.+1-1
}}
{% 33 %}5 Di{{"ga"+""}} {{
	.-30
}}3

_______________________
Raw block
_______________________

{% raw %}
raw
{%endraw%}
{% raw %}
	raw
{%endraw%}
	{% raw %}
		raw
	{%endraw%}
{% raw %}
{{ expression }}
{%endraw%}
{% raw %}
	{{ expression }}
{%endraw%}
{%raw%}
	{% code %}
	{# comment #}
{%endraw%}

=======================
