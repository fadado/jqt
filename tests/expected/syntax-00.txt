=======================
Syntax tests
=======================

_______________________
Comments
_______________________

1 prefixsuffix
2 prefix  suffix
3 prefix  suffix
4 prefix  suffix
5 
6 prefixsuffix
7 prefixsuffix
8 prefixsuffix
9 

_______________________
Expressions
_______________________

2 + 2 = 4
2 + 2 = 4
2 + 2 = 4

_______________________
One line blocks
_______________________

1 Diga 33
2 Diga 33
3 Diga 33
4 Diga 33
5 Diga 33

_______________________
Raw block
_______________________

raw
	raw
		raw
{{ expression }}
	{{ expression }}
	{% code %}
	{# comment #}

=======================
